sdk/scheduler/src/test/java/com/mesosphere/sdk/offer/evaluate/PortEvaluationStageTest.java

```diff
@@ -140,7 +140,7 @@ public void testPortOnHealthCheck() throws Exception {
 
         String taskName = "pod-type-0-" + TestConstants.TASK_NAME;
         PortEvaluationStage portEvaluationStage =
-                new PortEvaluationStage(desiredPorts, taskName, "test-port", 10000);
+                new PortEvaluationStage(desiredPorts, taskName, "port?test.port", 10000);
```

Comment:
Wait, I don't understand this one, either why it changed or the fact that that's not a valid environment variable name.